k8s/module/k8s

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+STATE="$1"
+FILES="$2"
+
+case "$STATE" in
+
+    NeedsArtifactReboot)
+        echo "No"
+    ;;
+
+    SupportsRollback)
+        echo "No"
+    ;;
+
+    ArtifactInstall)
+        find "$FILES/files" -name "*.yaml" -exec kubectl apply -f {} \;
```

Comment:
In the generator we are not enforcing that the files are `.yaml` (they could be `.yml` for instance). If should be fine here to apply all files, right? Regardless of the extension, they have been validated with the `kubectl --dry-run` at generation time.

Reply:
right, removed the need to have a yaml extension